packages/amplify_api/example/ios/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:Runner.xcodeproj">
+      location = "self:">
```

Comment:
Could you help me understand this change?

Reply:
Not too sure either.  Change happens automatically.  Seems to fix some Xcode build issue.

Ever since updating to Flutter 2.0 whenever I open and run the project for iOS this change is automatically done ... For this time I forgot to discard these changes.  

Looked online and found this here: https://stackoverflow.com/questions/66729236/flutter-ios-build-issues saying that it fixes the archive step for xcode.  If that's true we should probably include this change to all plugins instead of just api in the future.

Reply:
I have the same thing happen in my builds and agree it probably makes sense to include the change.

Reply:
ok we can make sure these changes are in the NS master branch.